bluesky/run_engine.py

```diff
@@ -2399,8 +2367,7 @@ def subscribe(self, func, name='all'):
             self._token_mapping[public_token] = private_tokens
             return public_token
 
-        if name not in DocumentNames:
-            name = DocumentNames[name]
+        name = DocumentNames[name]
```

Comment:
This condition didn't make sense before. Maybe we need to use the following?
```py
if name in DocumentNames:
    name = DocumentNames[name]
```

Reply:
This feature was deprecated before and it is removed from Python 3.8. The condition `name in DocumentNames` raises exception with the following message:
```
unsupported operand type(s) for 'in': 'str' and 'EnumMeta'
```
The condition was removed from v1.6.0 completely. I guess the purpose of this condition is to check if `name` is already of `DocumentName` enum type and perform the conversion if not. But the expected values of the 'name' parameter of the function are from the fixed list of strings, so we can perform conversion without verifying if it is a string. The exception will be raised if there is a problem.